### DISCUSSION TITLE
Included samples

### DISCUSSION THREAD

**Discussion by Nicole4Fox** [astarub/campus_app#100] [Q&A] [answered]
Why are currently the sample responses in the deployment branch?
https://github.com/astarub/campus_app/tree/deploy/test/pages

**Comment by henry-herrmann** [comment, marked as answer]
I wasn't around when they were written but as the deployment branch is just a copy of the master branch it was published with the PR that added these tests.

**Reply by Nicole4Fox** [reply]
i think long term we should clean the deployment branch from hacky development stuff like this

**Reply by henry-herrmann** [reply]
Yeah, that's another task that's yet to complete.

**Reply by Nicole4Fox** [reply]
I just noticed the same for unfinished modules with commented code in them. IMO thats what feature branches are for. Would it generally be possible to either clean out the branches and remove unused ones / to merge them or document them properly? Also i think it would be a good long term goal to free the main branch of hacky production code / code snippets / samples and such. Prime example is probably rubsignin_page.dart or one of the example mensa repsonses or the usage of print in production code. Maybe the maintainers can setup some contribution guideliens / general guidelines for this
